app/models/person.rb

```diff
@@ -359,11 +359,9 @@ def privacy_policy_accepted?
   alias privacy_policy_accepted privacy_policy_accepted?
 
   def privacy_policy_accepted=(value)
-    if %w(1 yes true).include?(value.to_s.downcase)
-      self.privacy_policy_accepted_at = Time.now.utc
-    else
-      self.privacy_policy_accepted_at = nil
-    end
+    self.privacy_policy_accepted_at = if %w(1 yes true).include?(value.to_s.downcase)
+                                        Time.now.utc
+                                      end
```

Comment:
Das ist ja nicht ganz dasselbe.. da die Tests hier nicht ausschlagen sollte das wohl i.O sein aber ich frage mich grad ob es einen Moment gibt in dem das attribut absichtlich auf `nil` gesetzt werden sollte :thinking: